install/operator/pkg/generator/assets/database/syndesis-db.yml.tmpl

```diff
@@ -212,11 +212,11 @@
         containers:
         - env:
           - name: POSTGRESQL_USER
-            value: {{.Syndesis.Spec.Components.Db.User}}
+            value: {{.Env.POSTGRESQL_USER}}
```

Comment:
Your going backwards here from what we want to do.

Reply:
I'm not familiar with what the team wants to do.

This was moved to an env to handle that an external database has the user defined in a secret, not in the CR spec. I did this to keep all of the information together in a secret and to avoid providing a password in the CR.

I agree that it's not ideal to have the divide between the two ways of deploying. So I'm happy to make changes to bring the "created" and "external" database logic branches closer together, if you have suggestions?

Reply:
The CR is still controlling the configuration and deployment of Syndesis.  
We just need the CR to say don't deploy the DB, just connect to an external one with the smallest amount of change possible.